config/Makefile

```diff
@@ -0,0 +1,376 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+# Allows for resources to be loaded from outside the root location of
+# the kustomize config file. Ensures that resource don't need to be
+# copied around the file system.
+#
+# See https://kubectl.docs.kubernetes.io/faq/kustomize
+#
+KOPTIONS := --load-restrictor LoadRestrictionsNone
+
+#
+# Include the main camel-k Makefile containing
+# basic common recipes like kustomize and vars
+# like VERSION
+#
+include script/Makefile
+
+#
+# Vars that can be overridden by external env vars
+#
+DEBUG ?= false
```

Comment:
Would `DRY_RUN` be more meaningful?

Reply:
This is literally a developer-type property that outputs the resource config to the screen, same as "-o yaml" in `kamel install`. Am not sure that `DRY_RUN` is more helpful in that context. If you feel it makes more sense then I can change it.

Reply:
Done